content/en/tutorial/10-signals.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Signals
+prev: /tutorial/09-error-handling
+next: /tutorial/11-links
+solvable: true
+---
+
+# Signals
```

Comment:
Not essential, but some more consistent line length in this document would be appreciated. Some lines wrap early, some go long; it's just a bit inconsistent when editing.